### PR TITLE
Docs fix type singleType findOne

### PR DESCRIPTION
**Pull request by fadhilx**
### What does it do?
fix doc typo for singleType in services.md, specifically at createOrUpdate section

### Why is it needed?

currently when i try to follow createOrUpdate, i got error `findOne is not a function`. so i know this is a typo because it related to #7721

### Related issue(s)/PR(s)
Issue:  #7718
PR: #7721

PR attempt: #8763

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
hi @fadhilx can you sign the CLA

**Review by derrickmehaffy — DISMISSED** [review]
LGTM

**Review by alexandrebodin — APPROVED** [review]
LGTM

File: docs/v3.x/concepts/services.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ module.exports = {
         model: 'restaurant',
         // if you are using a plugin's model you will have to add the `plugin` key (plugin: 'users-permissions')
       });
-      return this.findOne({ id: entry.id });
+      return this.find({ id: entry.id });
```
Comment (alexandrebodin) [inline docs/v3.x/concepts/services.md:346]: There is no id to pass here it should be a populate option or nothing :)

**Review by alexandrebodin — CHANGES_REQUESTED** [review]
The find method has a different param :)

File: docs/v3.x/concepts/services.md (hunk 1 above)
Comment (fadhilx) [reply docs/v3.x/concepts/services.md]: ok updated

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Comment by derrickmehaffy** [comment]
@fadhilx only thing pending is the CLA, there is an automated message at the top of the PR, can you click the button and sign that and I'll be able to merge.

**Comment by fadhilx** [comment]
@derrickmehaffy done

**Pull request merged**